sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -415,6 +520,9 @@ def get_gpu_price(row: pd.Series, spot: bool) -> Optional[float]:
 
 
 def _get_tpu_for_zone(zone: str) -> 'pd.DataFrame':
+    # Use hardcoded TPU V5 data as it is invisible in some zones.
+    if zone in TPU_V5_MISSING_ZONES_DF:
+        return TPU_V5_MISSING_ZONES_DF[zone]
```

Comment:
Will there be any other TPUs in the same zone? Should we still let the following code to run?

Reply:
Good point! Updated. Thanks!